src/directives/Onboard/CustomerQuote.tsx

```diff
@@ -1,26 +1,66 @@
-import { Box, useTheme } from '@mui/material';
+import { Box, useMediaQuery, useTheme } from '@mui/material';
+import { useIntl } from 'react-intl';
 import customerQuoteDark from 'images/customer_quote-dark.png';
 import customerQuoteLight from 'images/customer_quote-light.png';
-import { useIntl } from 'react-intl';
 
-interface Props {
-    hideQuote: boolean;
-}
-function CustomerQuote({ hideQuote }: Props) {
+function CustomerQuote() {
     const theme = useTheme();
     const intl = useIntl();
+    const belowMd = useMediaQuery(theme.breakpoints.down('md'));
 
-    if (hideQuote) {
+    if (belowMd) {
         return null;
     } else {
         return (
+            // TODO (customer quote) should switch this to HTML to load faster
```

Comment:
Worked on converting the image into HTML and did not want to lose that work. It looks _alright_ doing it with HTML. However, we'd need to add a new font/typography/something for the quote as having it render with the same fonts as other stuff was kind of weird.